src/routes/index.ts

```diff
@@ -0,0 +1,22 @@
+import bodyParser from "body-parser";
+import express from "express";
+import { getZodiac } from "../controllers/zodiac";
+import { getLuckyTextByRandomNumber } from "../controllers/lucky";
+import { loggingMiddleware } from "../middlewares/processing";
+
+export function setRoutes(app: express.Express): void {
+  try {
+    app.get("/", (req, res) => {
+      res.send("hello world");
+    });
+
+    // http://localhost:3000/zodiac?sign=leo
+
+    app.get("/zodiac", loggingMiddleware, getZodiac);
+
+    app.get("/lucky", loggingMiddleware, getLuckyTextByRandomNumber);
+    
+  } catch (error) {
+    console.log(error);
```

Comment:
You have handleErrors that is designed exactly for these cases.

Reply:
done